src/ducks/keyStore.ts

```diff
@@ -9,6 +9,9 @@ interface WalletKeyActionProps {
   privateKey?: string;
   keyType: KeyType;
   path?: string;
+  custom?: {
```

Comment:
a little confused by this field name. What does `custom` represent?

Reply:
Thanks for pointing that out! I'll add a comment.